src/frontend/Komponenter/Behandling/Totrinnskontroll/FatterVedtak.tsx

```diff
@@ -96,11 +96,11 @@ const FatterVedtak: React.FC<{
                     if (godkjent) {
                         hentBehandlingshistorikk.rerun();
                         hentTotrinnskontroll.rerun();
-                        settVisGodkjentModal(true);
```

Comment:
På linje 96 er det noe som ikke gir mening. For den vil jo alltid være "true".

Tror tanken var at hvis noe var underkjent så fikk man toast og gikk automatisk til oppgavebenken før...

Reply:
Men etter at vi gikk fra boolean til enum-verdi så fikk vi alltid "true" på  `if(godkjent)`


Så med mindre saksbehandlere ØNSKER en modal så kan denne PR-en endres til å kun endre linje 96:
`if(godkjent === Totrinnsresultat.GODKJENT)`

Reply:
Erstatt med https://github.com/navikt/familie-ef-sak-frontend/pull/1917